body-based-routing.Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN go mod download
 COPY cmd ./cmd
 COPY pkg ./pkg
 COPY internal ./internal
-WORKDIR /src/cmd/body-based-routing
+WORKDIR /src/cmd/bbr
 RUN go build -o /body-based-routing
```

Comment:
should this be "bbr" too?

Reply:
yes

Reply:
Same for the lines below this as well (28 & 30)

Reply:
done + changed the file name to bbr.Dockerfile (instead of body-based-routing.Dockerfile)